services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -3383,37 +3409,106 @@ private OfflinePushStatusInfo getOffLineJobStatus(
         leaderControllerUrl = controllerClient.getLeaderControllerUrl();
       } catch (VeniceException exception) {
         LOGGER.warn("Couldn't query {} for job status of {}", region, kafkaTopic, exception);
-        statuses.add(ExecutionStatus.UNKNOWN);
+        statuses.put(region, ExecutionStatus.UNKNOWN);
```

Comment:
Should we even query non-target regions ? just check the target regions statuses here?

Reply:
I think it's fine to query all regions since we can detect if non-targeted regions mistakenly ingest the message

Reply:
why do we want to do it ? It will throw unnecessarily exception `VeniceNoHelixResourceException` if we query non-existent topics.

Reply:
No, it doesn't throw. Since the push isn't created, the status will always be `ExecutionStatus.NOT_CREATED`.  [Ref](https://github.com/linkedin/venice/blob/664ba727bbf5edabd49c22dfcfc61b01b9eedbbf/services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/AbstractPushMonitor.java#L314)

Reply:
Discussed offline, we now query the targeted regions